docs/server-elasticsearch-setup.md

```diff
@@ -0,0 +1,81 @@
+---
+id: server-elasticsearch-setup
+title: How to set up Elasticsearch for the Temporal Server
+sidebar_label: Set up Elasticsearch
+---
+
+## Overview
+
+You can integrate [Elasticsearch](https://www.elastic.co/elasticsearch/) with the Temporal Server to enhance Workflow search functionality.
```

Comment:
do we have a list somewhere of WHAT functionality is enabled with ES? i feel like this is not discoverable until you dig into it. it would help people decide upfront whether or not adding ES is worth it.

Reply:
Ya, good question - the functionality will be described here: https://docs.temporal.io/docs/server-workflow-search
This page is purely meant to instruct on how to set up Elasticsearch - but I will link back to that Workflow search page from here.

Reply:
okiedokie